validphys2/src/validphys/dataplots.py

```diff
@@ -921,6 +923,106 @@ def plot_replica_sum_rules(pdf, sum_rules, Q):
     return fig
 
 
+@figure
+def plot_polarized_momentum(pdf, Q, xmin=0.001, angular_momentum=False):
+    """
+    Plot the correlated uncertainties for the truncated integrals of the polarized
+    gluon and singlet distributions.
+    """
+    from matplotlib.patches import Ellipse
```

Comment:
Why is this import here?

Reply:
Given that this is only used in polarized case, I thought it would be better if this is not imported at top level.

Reply:
It is better to import things at the top level, unless it is a specific library that you know can introduce some problems.